packages/toolkit/src/lib/vdp-sdk/model/types.ts

```diff
@@ -38,14 +40,24 @@ export type Model = {
   description: string;
   model_definition: string;
   configuration: {
-    [key: string]: any;
-  };
-  task: string;
-  state: ModelState;
+    [key: string]: string;
```

Comment:
@orangecoloured Do you think it's better to use Record<string, string> here?

Reply:
Technically, I see no difference. But `Record` is shorter and feels more "typed". I fixed it.